addons/onboarding/models/onboarding_progress_step.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.onboarding.models.onboarding_progress import ONBOARDING_PROGRESS_STATES
+
+
+class OnboardingProgressStep(models.Model):
+    _name = 'onboarding.progress.step'
+
+    _description = 'Onboarding Progress Step Tracker'
+
+    step_state = fields.Selection(
+        ONBOARDING_PROGRESS_STATES, string='Onboarding Step Progress', default='not_done')
+    company_id = fields.Many2one('res.company')
```

Comment:
Is it necessary ? And anyway if used, can probably be a related on progress_id.company_id ?